packages/shared/src/utils/isFeatureEnabled.tsx

```diff
@@ -4,13 +4,24 @@ import {
   isFeatureEnabledFragment$key,
   Role as RelayRole,
 } from '~/generated/isFeatureEnabledFragment.graphql';
-import { removeNullValues } from '~/shared/relay/removeNullValues';
+
+import { removeNullValues } from '../relay/removeNullValues';
 
 export enum FeatureFlag {
   KOALA = 'KOALA',
   MENTIONS = 'MENTIONS',
 }
 
+const PROD_FLAGS: Record<FeatureFlag, boolean> = {
+  KOALA: true,
+  MENTIONS: false,
```

Comment:
since Mentions are live for mobile already I'd say we don't need to put the web one behind a flag, can just deploy when it's ready!

Reply:
Make sense! Just remove the Mentions ff